apps/ledger-live-desktop/README.md

```diff
@@ -74,35 +74,11 @@ pnpm build:lld
 
 If you are using [Visual Studio Code](https://code.visualstudio.com/) IDE, we provide a [default debug configuration](https://github.com/LedgerHQ/ledger-live/tree/develop/.vscode/launch.json) that you can use to debug the main and renderer processes of the application.
 
-```json
```

Comment:
This content is already provided by the link above. No need to hard code it here as we'll never think about updating both the file and the readme